config/database.yml

```diff
@@ -7,6 +7,9 @@ default: &default
 
 development:
   <<: *default
+  username: postgres
```

Comment:
I would have hoped that these values would be configured through DATABASE_URL in `.env.development.local`, was this not the case? If we need it, this feels like a separate commit.

Reply:
No this is my misunderstanding! Will fix